src/views/search/components/AdvancedSearchModal/components/LabelsMultiSelect.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, useMemo } from 'react';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { labelParser } from '@kubevirt-utils/components/ListPageFilter/utils';
+import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { VirtualMachineModelGroupVersionKind } from '@kubevirt-utils/models';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { MultiTypeaheadSelect, MultiTypeaheadSelectOption } from '@patternfly/react-templates';
+import { OBJECTS_FETCHING_LIMIT } from '@virtualmachines/utils';
+
+type LabelsMultiSelectProps = {
+  labels: string[];
+  setLabels: React.Dispatch<React.SetStateAction<string[]>>;
+};
+
+const LabelsMultiSelect: FC<LabelsMultiSelectProps> = ({ labels, setLabels }) => {
+  const [vms] = useK8sWatchResource<V1VirtualMachine[]>({
+    groupVersionKind: VirtualMachineModelGroupVersionKind,
```

Comment:
no namespace? how will that work for non-admins that don't have cluster viewer permissions?